tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/api/ProgrammeMembershipResource.java

```diff
@@ -148,7 +170,6 @@ public ResponseEntity<List<ProgrammeMembershipDTO>> getAllProgrammeMemberships(
   @GetMapping("/programme-memberships/{id}")
   @PreAuthorize("hasPermission('tis:people::person:', 'View')")
   public ResponseEntity<ProgrammeMembershipDTO> getProgrammeMembership(@PathVariable String id) {
```

Comment:
Unless there is a good reason for the change, probably best to leave the debug logging 😁